packages/cra-template/template/src/Components/ResultList.tsx

```diff
@@ -23,7 +23,7 @@ interface ResultListProps {
 
 function ListItemLink(props: any) {
   return (
-    <ListItem style={{padding: 0}} component="a" {...props}>
+    <ListItem style={{padding: 0}} component="a" {...(props as unknown)}>
```

Comment:
I think this function could be typed as: 

```
import {ListItemProps} from "@material-ui/core";

function ListItemLink(props: ListItemProps<"a">) {
  return <ListItem {...props} button component="a" />;
}
```
and we could remove the `any/unknown` cast